assets/freedesktop/org.zealdocs.zeal.appdata.xml.in

```diff
@@ -1,10 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop">
   <id>org.zealdocs.zeal</id>
+  <launchable type="desktop-id">org.zealdocs.zeal.desktop</launchable>
   <name>Zeal</name>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-3.0-or-later</project_license>
-  <developer_name>Oleg Shparber</developer_name>
+  <developer id="zealdocs.org">
```

Comment:
@trollixx Let me know if you have a personal, different domain that you prefer to use here.

And here's the [documentation](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-developer) about this tag, if you're interested in learning more about it.

Reply:
Shouldn't ID be `org.zealdocs`? Per the linked docs:
> It is recommended to use a reverse-DNS name, like org.gnome or io.github.ximion, or a Fediverse handle (like @user@example.org) as ID to achieve a higher chance of uniqueness.

Reply:
Indeed! Good catch, thanks.